viewing_party/utils_01.py

```diff
@@ -0,0 +1,24 @@
+def create_movie(movie_title, genre, rating):
+    if movie_title and genre and rating: 
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.
 
```suggestion
    if not movie_title and not genre and not rating:
        return None
        
    movie = {
            'title': movie_title,
            'genre': genre,
            'rating': rating
        }
        return movie
```